sdk/storage/inc/blobs/internal/protocol/blob_rest_client.hpp

```diff
@@ -7,27 +7,88 @@
 #include <string>
 #include <vector>
 
-#include "common/xml_wrapper.hpp"
 #include "common/internal/protocol/rest_client_utility.hpp"
+#include "common/xml_wrapper.hpp"
 
 namespace Azure
 {
 namespace Storage
 {
 namespace Blobs
 {
+
+enum class AccessTier
+{
+  Unknown,
+  P1,
+  P2,
+  P3,
+  P4,
+  P6,
+  P10,
+  P15,
+  P20,
+  P30,
+  P40,
+  P50,
+  P60,
+  P70,
+  P80,
+  Hot,
```

Comment:
I think page blob tier should be distinguished with block blob tier.

Reply:
.Net Track2 doesn't distinguish them.